std/algorithm/sorting.d

```diff
@@ -145,20 +145,20 @@ Checks whether a forward range is sorted according to the comparison
 operation $(D less). Performs $(BIGOH r.length) evaluations of $(D
 less).
 
-Unlike $(LREF isSorted), $(LREF isStrictlyMonotonic) does not allow for equal values,
+Unlike isSorted, isStrictlyMonotonic does not allow for equal values,
```

Comment:
Oh I see what you mean. Yes, it would be best if whatever is highlighted would come in code font.